matchbox_socket/src/webrtc_socket/native.rs

```diff
@@ -57,6 +58,10 @@ impl Signaller for NativeSignaller {
                         } else {
                             *attempts -= 1;
                             warn!("connection to signaling server failed, {attempts} attempt(s) remain");
+                            warn!("waiting 3 seconds to re-try connection...");
+                            Delay::new(Duration::from_secs(3)).await;
```

Comment:
Probably not necessary at the scale we're working with but might be worth adding some jitter to avoid correlated overloads. Might even be worth pulling in a crate like `retry` to handle this loop

Reply:
In a future iteration, I think this is a good idea. For now I think this is ok :)